lib/error.js

```diff
@@ -31,7 +31,7 @@ var ERRNO = {
   REQUEST_BLOCKED: 125,
   ACCOUNT_RESET: 126,
   INVALID_UNBLOCK_CODE: 127,
-  MISSING_TOKEN: 128,
+  // MISSING_TOKEN: 128,
```

Comment:
Doesn't look like this is used anywhere, did you intend to keep it around in case we bring back missing token somewhere else?